contracts/tonomy/src/tonomy.cpp

```diff
@@ -207,10 +190,57 @@ namespace tonomysystem
       account_type_table account_type(get_self(), get_self().value);
       account_type.emplace(get_self(), [&](auto &row)
                            {
-         row.account_name = random_name;
+         row.account_name = account_name;
          row.acc_type = enum_account_type::App;
          row.version = 1; });
    }
+   
+
+   void tonomy::newapp(
```

Comment:
```cpp
void tonomy::newapp(
      eosio::name owner,
       string app_name,
       string description,
       checksum256 username_hash,
       string logo_url,
       string origin
   {
      eosio::require_auth({owner, get_self()}); // this will successfully use the SSO permission

      create_app(owner, app_name, description, username_hash, logo_url, origin);
   }

   void tonomy::newappadmin(
      name owner,  
      string app_name,
      string description,
      checksum256 username_hash,
      string logo_url,
      string origin)
{
      require_auth(native::governance_name); // check authorization is gov.tmy

      create_app(owner, app_name, description, username_hash, logo_url, origin);
   }

```

Reply:
not sure why, but this documentation does not seem to be updated yet. know why?
https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/pull/340/files#diff-f2e5fca1239a412f0833e52061be1121ff156ddb471d3fe37065f1efac632f81R77